### DISCUSSION TITLE
Flexible cross referencing: allow figures to be labelled as tables

### DISCUSSION THREAD

**Discussion by LibrEars** [quarto-dev/quarto-cli#3914] [Feature Requests]
Hi all =)

In some use-cases like writing a book, thesis or review one would like to add very large tables with custom formatting (split cells, smaller margins, colors, ...). So it might be sometimes convenient to create them with a third-party tool like libreOffice.

Would it be possible to add tables as vector-graphics (svg, pdf...) in the markdown document and cross reference them with `@tbl-table1` (for HTML and PDF output):

```{yml}
project:
  type: book

...

format:
  html:
    default-image-extension: svg
  pdf:
    default-image-extension: pdf
```

E.g. chapter1.md
```{markdown}
This awesome overview over the literature can be seen in @tbl-super_fancy

![Some very huge table with custom format](./Tables/super_fancy_table){#tbl-super_fancy}
``` 

this would introduce a somehow simple workflow

**Comment by cscheid** [comment]
We'll be reworking the crossref system in the near future, and we hope something like this will be possible.

**Reply by dmenne** [reply]
Please, I am looking into the pulls each morning.  We decided not to use Quarto for a long-term new project, because currently cross-referencing and figure captions are only half-heartedly supported. It was a mess in knitr/rmarkdown (see https://gist.github.com/dmenne/f8eb291c9e71a5de44764d442e8bdefd), but somehow one could work around, and I had hoped work-around solutions can be avoided with Quarto.

**Reply by LibrEars** [reply]
I agree that workarounds should be avoided. This idea does not touch the default support of clean and simple tables that are already working well and are in line with the simplistic markdown language. 

The proposal suggest to add an _extended support_ for more sophisticated tables that are hard to be realized in clean markdown (without some bloated and complex syntax expansion?). But maybe there are better ways to accomplish this without sacrificing too much simplicity :)

**Reply by cscheid** [reply]
@dmenne We hear you, and appreciate the enthusiasm and feedback.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/4944
